src/main/scala/net.manub.embeddedkafka/schemaregistry/config.scala

```diff
@@ -1,39 +1,104 @@
 package net.manub.embeddedkafka.schemaregistry
 
+import java.nio.file.{Files, Path, Paths}
+import java.nio.file.StandardCopyOption.REPLACE_EXISTING
+
+import io.confluent.kafka.schemaregistry.avro.AvroCompatibilityLevel
 import net.manub.embeddedkafka.{
   EmbeddedKafkaConfig => OriginalEmbeddedKafkaConfig
 }
+import org.apache.kafka.clients.CommonClientConfigs.SECURITY_PROTOCOL_CONFIG
+import org.apache.kafka.clients.consumer.ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG
+import org.apache.kafka.common.config.SslConfigs._
 
 trait EmbeddedKafkaConfig extends OriginalEmbeddedKafkaConfig {
```

Comment:
This is a good point, and I can see that it could be controversial.

Personally, I agree with the original author that it's ok to just use `EmbeddedKafkaConfig`. The reason being:
- This library itself is made for integrating schema registry as can be seen in the name `embedded-kafka-schema-registry`, there is no need to tell the user twice.
- From usage's perspective, as long as the user uses `EmbeddedKafkaStreamsAllInOne` trait, it will all work. The name of the config trait doesn't really matter nor it will be confusing to the users.
- From our perspective, SSL should always be used in stream testing by default (just like schema compatibility setting), so no need to make it special too.

This is just my personal thoughts, happy to hear from others.

Reply:
Sounds good